frontend/src/Components/Page/Sidebar/PageSidebar.js

```diff
@@ -49,7 +49,7 @@ const links = [
     to: '/settings',
     children: [
       {
-        title: translate('Indexers'),
+        title: translate('IndexerProxies'),
```

Comment:
This is meant to be Indexers to house other global indexer settings. This discussion was had on Discord

Reply:
got it